qa/workunits/osdc/object_cacher_misc.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -ex
+
+ceph_test_objectcacher_misc --flush-test > /dev/null 2>&1
```

Comment:
Does this test produce a lot of output?  Having the output available in `teuthology.log` is useful in case of failure:
```suggestion
ceph_test_objectcacher_misc --flush-test
```

Reply:
few output, good